code/menuui/mainhallmenu.cpp

```diff
@@ -1978,6 +1982,7 @@ void misc_anim_init(main_hall_defines &m, bool first_time, int base_num)
 		m.misc_anim_delay.back().push_back(-1);
 		m.misc_anim_delay.back().push_back(0);
 		m.misc_anim_delay.back().push_back(0);
+		m.misc_anim_initial_delay.push_back(-1);
```

Comment:
Add a comment here because "0 for the others" might lead someone to think this should be 0.

Reply:
done.